pkgs/development/python-modules/future-stubs/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi, future }:
+
+buildPythonPackage rec {
+  pname = "future-stubs";
+  version = "0.18.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "ae42148268d32d25643bd0f985d528924d17c2830b49a13a5b289e22878ca000";
+  };
+
+  propagatedBuildInputs = [ future ];
+
+  # As packaged in pypi, the future-stubs library had several minor but fatal
+  # packaging problems. These patches fix the problems:
+  # - setup.cfg references a LICENSE.txt file that doesn't exist.
+  # - setup.py install_requires "python-future" instead of "future"
+  # - The package is missing the `py.typed` files that mypy requires.
```

Comment:
These can probably be fixed by checkout on the source repo https://github.com/PythonCharmers/python-future